common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -61,7 +61,7 @@ private void initOverlays() {
             try {
                 Overlay overlay = (Overlay) FieldUtils.readField(overlayField, this, true);
                 OverlayInfo annotation = overlayField.getAnnotation(OverlayInfo.class);
-                OverlayManager.registerOverlay(overlay, annotation);
+                OverlayManager.registerOverlay(overlay, annotation, shouldEnable);
```

Comment:
Am I being anal if I think you can call `this.canEnable()` here instead of passing it as an argument?

Reply:
Then I think we should not pass instance variables as arguments to private helper methods. :-)